react/src/Header.js

```diff
@@ -103,7 +103,7 @@ const MainMenu = ({ header }) => {
                 <MenuItem path="/users" label="Users" />
                 <MenuSeparator />
                 <MenuItem path="/env" label="Environments" />
-                <MenuItem path="/accounts" label="Accounts" />
+                <MenuItem path="/accounts?all=true" label="Accounts" />
```

Comment:
Maybe a comment reminding people like me what `?all=true` does here?

Reply:
Got it